api-reference/_hidden/dxPieChartCommonAnnotationConfig/series.md

```diff
@@ -5,7 +5,29 @@ default: undefined
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Anchors the annotation to a series point. Accepts the [name](/Documentation/ApiReference/Data_Visualization_Widgets/dxPieChart/Configuration/series/#name) of the point's series.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Use this option when the annotation is positioned relative to an [argument](/api-reference/_hidden/BaseChartAnnotationConfig/argument.md '{basewidgetpath}/Configuration/annotations/#argument').
+
+For example, the following **PieChart** fragment displays two series and an annotation. The annotation is positioned relative to the argument `B` but not anchored to any of the two series points that correspond to this argument. To anchor the annotation to a series point, specify the annotation's **series** option:
```

Comment:
Illustrations are needed.

Reply:
Seems like this part should not be included since you cannot create annotations in PieChart that are not anchored to a series point.